scripts/travisci_run.sh

```diff
@@ -17,6 +17,9 @@ export BUCK_NUM_THREADS=3
 # Make sure that everything builds in case a library is not covered by a test.
 ./bin/buck build --num-threads=$BUCK_NUM_THREADS src/... test/...
 
+# Run error prone checks
+./bin/buck build --config sanitizers.error_prone=1 --num-threads=$BUCK_NUM_THREADS src/... test/...
```

Comment:
The reason I did it was because there were errors in error prone checks for test/... . But I agree, let us do that and fix the breakage before submitting this.